.appveyor.yml

```diff
@@ -27,6 +27,7 @@ install:
 
 # Test scripts
 test_script:
+  - python qcodes\tests\setup_config.py
```

Comment:
Is this still needed?

Reply:
I suppose it shouldn't be, no. Let me try to remove it.